src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,8 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+object LetterRemover {
```

Comment:
Please check task requirements. You need to use companion object.
If you have some issue with code validation plugin you can try to use private constructor or in this particular case ignore it if code returns required result